code/__DEFINES/tgs.dm

```diff
@@ -95,8 +95,13 @@
 #define TGS_EVENT_WATCHDOG_SHUTDOWN 15
 /// Before the watchdog detaches for a TGS update/restart. No parameters.
 #define TGS_EVENT_WATCHDOG_DETACH 16
-// We don't actually implement this value as the DMAPI can never receive it
+// We don't actually implement these 3 events as the DMAPI can never receive them.
```

Comment:
```suggestion
// We don't actually implement these 4 events as the DMAPI can never receive them.
```

Isn't it 4?

Reply:
Go pester Cyberboss about this; DMAPI is managed by him.